javalin/src/main/java/io/javalin/http/servlet/JavalinServletContext.kt

```diff
@@ -52,7 +52,9 @@ data class JavalinServletContextConfig(
     val defaultContentType: String,
     val jsonMapper: JsonMapper,
     val strictContentTypes: Boolean,
-    val multipartConfig: MultipartConfig
+    val multipartConfig: MultipartConfig,
+    val eventManager: io.javalin.event.EventManager,
+    val asyncExecutor: Lazy<io.javalin.http.util.AsyncExecutor>
```

Comment:
I think EventManager should keep executor internally, no need to pass it like that here.